IyiKuyoro/Tiddy
# Add default slash command

# Add default slash command

## What does this PR do

This PR adds the default slash command

## Description of Task to be completed

- [x] Add a controller for slash commands

## How should this be manually tested

- Follow the instructions on the HOW_TO_CONTRIBUTE file.
- Change all environment variables to your actual variables.
- Build the production code using `npm run build`.
- Start the server using `npm run dev` ensure you have added the latest migrations first.
- Setup your bot and install.
- Try `/tiddy`

## What are the relevant PT board stories link
#166484210

## Any background context you want to add
